scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -561,6 +564,41 @@ static u64 sched_prio_to_latency_weight(u64 prio)
 	return sched_prio_to_weight[DL_MAX_LAT_PRIO - prio - 1];
 }
 
+static void do_iowait_boost(struct task_struct *p, struct task_ctx *taskc, s32 cpu)
+{
+	if (!iowait_boost || !p || !taskc || cpu < 0)
+		return;
+	if (!(p->flags & PF_KTHREAD) || (p->flags & TASK_IDLE))
+		return;
+
+	if (BPF_CORE_READ_BITFIELD_PROBED(p, in_iowait)) {
+		u64 now = bpf_ktime_get_ns();
+		if (taskc->boost_ctx.deadline == 0)
+			taskc->boost_ctx.deadline = now +
+				iowait_boost_max_dur_ms * NSEC_PER_MSEC;
+
+		if (now > taskc->boost_ctx.deadline) {
+			taskc->boost_ctx.curr_ios = 0;
+			taskc->boost_ctx.deadline = 0;
+			return;
+		}
+
+		taskc->boost_ctx.curr_ios++;
```

Comment:
So IIUC, this field tracks the number of scheduling events that happen while `p` is in `in_iowait` state? If so, could we rename the variable to something like `io_requests`?